app/controllers/api/projects.php

```diff
@@ -607,6 +607,37 @@
         $response->dynamic($project, Response::MODEL_PROJECT);
     });
 
+App::patch('/v1/projects/:projectId/auth/password-dictionary')
+    ->desc('Update Project users limit')
+    ->groups(['api', 'projects'])
+    ->label('scope', 'projects.write')
+    ->label('sdk.auth', [APP_AUTH_TYPE_ADMIN])
+    ->label('sdk.namespace', 'projects')
+    ->label('sdk.method', 'updateAuthLimit')
+    ->label('sdk.response.code', Response::STATUS_CODE_OK)
+    ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
+    ->label('sdk.response.model', Response::MODEL_PROJECT)
+    ->param('projectId', '', new UID(), 'Project unique ID.')
+    ->param('enabled', false, new Boolean(true), 'Set whether or not to enable checking user\'s password against most commonly used passwords. Default is false.')
+    ->inject('response')
+    ->inject('dbForConsole')
+    ->action(function (string $projectId, bool $enabled, Response $response, Database $dbForConsole) {
+
+        $project = $dbForConsole->getDocument('projects', $projectId);
+
+        if ($project->isEmpty()) {
+            throw new Exception(Exception::PROJECT_NOT_FOUND);
+        }
+
+        $auths = $project->getAttribute('auths', []);
+        $auths['passwordDictionary'] = (bool) filter_var($enabled, FILTER_VALIDATE_BOOLEAN);
```

Comment:
We are missing migration for this. No need for collection migration, but w eneed to update each project document and add default value for passwordDictionary

Reply:
default is false, which we make sure to test for null when reading so it will work without migration. You think we need to?

Reply:
Not sure what our strategy here is, let's maybe confirm with Eldad.
My call would be to have migration. In future someone might want to use the attribute but won't know there **has to** be default specified in `getAttribute`, and it can cause a bug.
Having migration ensures the attribute is always set, so default can be set on database-level, not in the controller code.

Reply:
Updated migration